dashboard/app/models/script.rb

```diff
@@ -678,7 +678,15 @@ def has_standards_associations?
   end
 
   def standards
-    stages.map(&:standards).flatten.uniq.map(&:summarize)
+    standards = stages.map(&:standards).flatten.uniq
+    standards_with_stages = []
+    standards.each do |standard|
+      standards_summary = standard.summarize
+      stages_by_standard = stages && standard.stages
```

Comment:
It looks like the intersection operator is a single ampersand: https://ruby-doc.org/core-2.4.1/Array.html#method-i-26

```suggestion
      stages_by_standard = stages & standard.stages
```